deployment/sriov-network-operator/templates/certificate.yaml

```diff
@@ -0,0 +1,72 @@
+{{- if and .Values.operator.admissionControllers.enabled }}
```

Comment:
do we need the "and" here ?

Reply:
Good catch, missed that. Will address once I get some more feedback on the PR.